libcontainer/standard_init_linux.go

```diff
@@ -142,12 +142,6 @@ func (l *linuxStandardInit) Init() error {
 			return errors.Wrap(err, "set nonewprivileges")
 		}
 	}
-	// Tell our parent that we're ready to Execv. This must be done before the
-	// Seccomp rules have been applied, because we need to be able to read and
-	// write to a socket.
```

Comment:
Is this still a concern? If not, please amend the commit message to explain why.

Reply:
I don't remember why syncParentReady was moved up. During syncParentReady, the parent process just sets rlimits for the child and runs the PreStart OCI hook, so I don't think it matters if it is before or after the seccomp. I'm moving it back where it was and it seems to work fine.

<details>

I also ran some tests with blocking the write syscall: In the case noNewPrivileges==false, blocking the "write" syscall in the seccomp profile was already broken on master because we had `unix.Write(fd, []byte("0"))` after `seccomp.InitSeccomp` anyway. In practice, it breaks with the following symptoms:
```
$ sudo ../runc run c1
ERRO[0000] cannot start an already running container
```
(the parent process misinterpret the empty data on the pipe, so the error message is confusing)

I would argue that blocking the read/write syscalls in the seccomp profile is a bad idea because:
- The read/write syscalls are needed in all use cases I can think of
- They are part of the 4 syscalls initially accepted in the strict mode of seccomp (see manual page seccomp(2))
- runc was already broken when blocking "write" (at least with noNewPrivileges==false) so probably nobody is doing that.

So if we ever need to move it, that should be fine too.

</details>